### ISSUE TITLE
[Fix] Modal, on Profile Page View, does not read out from screen reader

### ISSUE THREAD

**Issue by SukhsimranpreetSekhon** [CDH-Studio/UpSkill#1162]
**Describe the bug**
- Modal, on Profile Page View, that opens after selecting toggle as "Public" does not read out it's contained information through a screen reader such as NVDA.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to 'Profile Page View'.
2. Click on 'toggle dropdown' for a card.
3. Switch option to 'Public' in the dropdown.
4. Hear screen reader not read out the modal's contained information to the user.

**Expected behaviour**
- The modal should read out the information to the user after selecting "Public" as a toggle dropdown option.

**Screenshots**

**Additional context**
